dali/pipeline/operators/geometric/bb_flip.cc

```diff
@@ -75,7 +75,7 @@ void BbFlip<CPUBackend>::RunImpl(dali::SampleWorkspace *ws, const int idx) {
   auto output_data = output.mutable_data<float>();
 
   for (int i = 0; i < input.size(); i += 4) {
-    auto bbox = ltrb_ ? BoundingBox::FromLtrb(&input_data[i])
+    auto bbox = ltrb_ ? BoundingBox::FromLtrb(&input_data[i], BoundingBox::NoBounds())
                       : BoundingBox::FromXywh(&input_data[i]);
```

Comment:
Why the double standard here? It's more likely that an XYWH box will be out of bounds than LTRB due to rounding problems.

Reply:
Done